src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/executors/Executors.scala

```diff
@@ -101,6 +101,7 @@ class Executors @Inject()(dagInstanceRepository: DagInstanceRepository, jobInsta
   }
 
   private def updateJob(jobInstance: JobInstance): Future[Unit] = {
+    logger.info(s"Job updated. ID = ${jobInstance.id} STATUS = ${jobInstance.jobStatus} EXECUTOR_ID = ${jobInstance.executorJobId}")
```

Comment:
Slightly off-topic, but maybe we could add a second parameter `oldJobInstance` and only update if the entity has changed?